fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -647,6 +647,24 @@ public void testGetChildrenWithBinary() {
     }
 
     @Test
+    public void testGetChildrenWithBinaryCustomProperties() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Container container = containerService.findOrCreate(session, "/" + pid);
+        final FedoraResource resource = binaryService.findOrCreate(session, "/" + pid + "/a");
+        final javax.jcr.Node node = resource.getNode();
+        node.setProperty("dc:title", "this-is-some-title");
+
+        final Graph graph =
+                container.getChildren().next().getTriples(subjects, PropertiesRdfContext.class).asModel().getGraph();
```

Comment:
I know. You're assuming that the child will be the _first_ child of the container, and it now is, but it may or may not always be-- you're assuming that we will never change the code to add other children in some other way in between your method calls and that the JCR iteration order won't ever change. I'm not saying your test doesn't work now, I'm saying it's a bit brittle unless you specify which child to retrieve.